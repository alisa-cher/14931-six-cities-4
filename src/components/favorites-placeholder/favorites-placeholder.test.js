import React from 'react';
import renderer from 'react-test-renderer';
import FavoritesPlaceholder from "./favorites-placholder.jsx";

it(`Favorites placeholders renders correctly`, () => {
  const tree = renderer
    .create(
        <FavoritesPlaceholder
          heading={`some-heading`}
          title={`some-title`}
          description={`some-description`}
        />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
