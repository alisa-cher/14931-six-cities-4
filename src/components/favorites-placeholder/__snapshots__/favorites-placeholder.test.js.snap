// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites placeholders renders correctly 1`] = `
<section
  className="favorites favorites--empty"
  style={
    Object {
      "minHeight": "550px",
    }
  }
>
  <h1
    className="visually-hidden"
  >
    some-heading
  </h1>
  <div
    className="favorites__status-wrapper"
  >
    <b
      className="favorites__status"
    >
      some-title
    </b>
    <p
      className="favorites__status-description"
    >
      some-description
    </p>
  </div>
</section>
`;
